### PR TITLE
Delete an item from a list (inject, atomic testing, scenario, simlation)

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Related issues

* https://github.com/OpenBAS-Platform/openbas/issues/1205

### CODE REVIEW

File: openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingHeader.tsx [hunk 1]
```diff
@@ -91,9 +105,8 @@ const AtomicTestingHeader = () => {
         }
         <AtomicTestingPopover
           atomic={injectResultDto}
-          actions={['Duplicate', 'Update', 'Delete']}
-          openEdit={openEditId === injectResultDto.inject_id}
-          setOpenEdit={setOpenEditId}
+          actions={['Update', 'Duplicate', 'Delete']}
+          onDelete={() => navigate('/admin/atomic_testings')}
```
Comment (savacano28) [inline openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingHeader.tsx:109]: Here we could filter is an inject doesnt have a contract also.

File: openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingPopover.tsx [hunk 2]
```diff
@@ -1,133 +1,90 @@
...
-    if (actions.includes('Update')) {
-      entries.push({ label: 'Update', action: () => (isNotEmptyField(setOpenEdit) ? setOpenEdit(atomic.inject_id) : setEdition(true)) });
-    }
-  }
-  if (actions.includes('Delete')) entries.push({ label: 'Delete', action: () => setDeletion(true) });
+  // Button Popover
+  const entries = [];
+  if (actions.includes('Duplicate')) entries.push({ label: 'Duplicate', action: () => handleOpenDuplicate() });
+  if (actions.includes('Update')) entries.push({ label: 'Update', action: () => handleOpenEdit() });
+  if (actions.includes('Delete')) entries.push({ label: 'Delete', action: () => handleOpenDelete() });
```
Comment (savacano28) [inline openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingPopover.tsx:61]: Or here, but we need to filter update and duplicate when the inject doesn't have a contract.

**Comment by savacano28** [comment]
Tested OK, just need to add a filter for options on atomic without contracts.

**Comment by RomuDeuxfois** [comment]
@savacano28 I made changes
Can you confirm it suit your idea ?

**Comment by RomuDeuxfois** [comment]
@savacano28 can tou approve if it's ok for you?
Thank you

**Review by savacano28 — APPROVED** [review]

**Pull request merged**